src/model/article-vor.v1.yaml

```diff
@@ -5,6 +5,11 @@ allOf:
   - $ref: ../snippets/article-vor.v1.yaml
   - $ref: ../misc/article.v1.yaml
   - properties:
+        abstract:
+            type: object
+            properties:
```

Comment:
I may be wrong, but it looks like the `doi` property itself remains optional in an abstract, and can the `doi` property just move to the `src/misc/abstract.v1.yaml` file? That may imply a PoA abstract can also have an optional `doi` property, which doesn't seem too much of a stretch of the rules. Then, wouldn't this file get its abstract from the `src/misc/article.v2.yaml ` file referenced?

Reply:
I would like to retain the same rules for the abstract when it is present so I'm not going to introduce this now. It may be worth considering but I don't want to question the decision to not make doi optional for POA as part of this PR. Although the diff is a bit bigger than it need be if I made this adjustment it would make it, it's not substantial.

Thanks for your feedback.